cmd/commands/connection/util.go

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (C) 2020 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package connection
+
+import (
+	"fmt"
+	"math/big"
+	"time"
+
+	"github.com/mysteriumnetwork/node/money"
+	"github.com/mysteriumnetwork/node/tequilapi/contract"
+)
+
+func printProposal(p *contract.ProposalDTO) {
+	msg := fmt.Sprintf("| Identity: %s | Type: %s | Country: %s | Price: %s/min   %s/GB |",
+		p.ProviderID,
+		p.ServiceDefinition.LocationOriginate.NodeType,
+		p.ServiceDefinition.LocationOriginate.Country,
+		pricePerMinute(p.PaymentMethod),
+		pricePerGB(p.PaymentMethod),
+	)
+	fmt.Println(msg)
```

Comment:
```suggestion
	fmt.Printf("| Identity: %s | Type: %s | Country: %s | Price: %s/min   %s/GB |\n",
		p.ProviderID,
		p.ServiceDefinition.LocationOriginate.NodeType,
		p.ServiceDefinition.LocationOriginate.Country,
		pricePerMinute(p.PaymentMethod),
		pricePerGB(p.PaymentMethod),
	)
```

Reply:
Dont need the new line. We already do `Println`